zuul.d/whitebox_neutron_tempest_jobs.yaml

```diff
@@ -0,0 +1,149 @@
+---
+# It contains the list of jobs running against
+# whitebox-neutron-tempest-plugin rdo third party check job
+
+- job:
+    name: whitebox-neutron-tempest-plugin-podified-multinode-edpm-deployment-crc-2comp
+    parent: podified-multinode-edpm-deployment-crc-2comp
+    description: |
+      A multinode EDPM Zuul job which one controller, one extracted crc and
+      2 computes. This job will run with meta content provider to test
+      whitebox-neutron-tempest-plugin opendev patches. It will validate the
+      deployment by running whitebox-neutron-tempest-plugin tests.
+    vars:
+      cifmw_run_test_role: test_operator
+      cifmw_os_must_gather_timeout: 28800
+      cifmw_test_operator_timeout: 14400
+      cifmw_block_device_size: 40G
+      cifmw_test_operator_concurrency: 6
+      cifmw_test_operator_tempest_network_attachments:
+        - ctlplane
+      cifmw_test_operator_tempest_container: openstack-tempest-all
+      cifmw_test_operator_tempest_registry: "{{ content_provider_os_registry_url | split('/') | first }}"
+      cifmw_test_operator_tempest_namespace: "{{ content_provider_os_registry_url | split('/') | last }}"
+      cifmw_test_operator_tempest_image_tag: "{{ content_provider_dlrn_md5_hash }}"
+      cifmw_test_operator_tempest_extra_images:
+          # TODO(chandan): Replace rocky qcow2 url once rhos-ops hosts it
+        - URL: "https://chandankumar.fedorapeople.org/rocky9_latest_neutron_whitebox.qcow2"
+          name: custom_neutron_guest
```

Comment:
It seems that should be added to fix image issue as Lukas suggested in OSPTH-8989:
    name: custom_neutron_guest
    diskFormat: "qcow2" <--- HERE